jwst/assign_wcs/util.py

```diff
@@ -506,7 +506,8 @@ def get_object_info(catalog_name=None):
 def create_grism_bbox(input_model,
                       reference_files,
                       mmag_extract=99.0,
-                      extract_orders=None):
+                      extract_orders=None,
+                      extract_height=None):
```

Comment:
If the keyword is `extract_height` it would be clearer if it was the full height of the extraction.  And this is already what the current keyword is.

https://github.com/spacetelescope/jwst/blob/ff9ea4017ebdc62309cf97661a5093f72c402722/jwst/extract_2d/extract_2d.py#L37-L40

If it is to be the half height, then `extraction_half_height` would be clearer to users I think.